app/gui2/src/stores/graph/index.ts

```diff
@@ -664,10 +671,11 @@ export const { injectFn: useGraphStore, provideFn: provideGraphStore } = createC
     }
 
     /** Iterate over code lines, return node IDs from `ids` set in the order of code positions. */
-    function pickInCodeOrder(edit: MutableModule, ids: Set<NodeId>): NodeId[] {
+    function pickInCodeOrder(ids: Set<NodeId>): NodeId[] {
+      assert(syncModule.value != null)
+      const func = unwrap(getExecutedMethodAst(syncModule.value))
+      const lines = func.body instanceof Ast.BodyBlock ? func.body.lines : []
```

Comment:
`Function` has `bodyExpressions` that does the right thing if the function is single-line.